pkg/backup/backup/backup_manager.go

```diff
@@ -665,11 +659,50 @@ func (bm *backupManager) makeBRBackupJob(backup *v1alpha1.Backup) (*batchv1.Job,
 	return job, "", nil
 }
 
-func (bm *backupManager) tryBackupIfCanSnapshot(b *v1alpha1.Backup, tc *v1alpha1.TidbCluster) (string, error) {
+// save cluster meta to external storage since k8s size limitation on annotation/configMap
+func (bm *backupManager) saveClusterMetaToExternalStorage(b *v1alpha1.Backup, csb *snapshotter.CloudSnapBackup) (string, error) {
+
+	data, err := json.Marshal(csb)
+	if err != nil {
+		return "ParseCloudSnapshotBackupFailed", err
+	}
+	ctx, cancel := backuputil.GetContextForTerminationSignals(b.ClusterName)
```

Comment:
Why need to listen to the termination signal?

Reply:
fix in https://github.com/pingcap/tidb-operator/pull/4784/commits/d37ccf3973fcd2463ece589df5ffa2f9a46b0585